README.md

```diff
@@ -1,3 +1,72 @@
-# jellyfin-plugin-ldapauth
+<h1 align="center">Jellyfin LDAP-Auth Plugin</h1>
+<h3 align="center">Part of the <a href="https://jellyfin.media">Jellyfin Project</a></h3>
+
+<p align="center">
+
+<img alt="Logo Banner" src="https://raw.githubusercontent.com/jellyfin/jellyfin-ux/master/branding/SVG/banner-logo-solid.svg?sanitize=true"/>
+<br/>
+<br/>
+<a href="https://github.com/jellyfin/jellyfin-plugin-ldapauth/actions?query=workflow%3A%22Test+Build+Plugin%22">
+<img alt="GitHub Workflow Status" src="https://img.shields.io/github/workflow/status/jellyfin/jellyfin-plugin-ldapauth/Test%20Build%20Plugin.svg">
+</a>
+<a href="https://github.com/jellyfin/jellyfin-plugin-ldapauth">
+<img alt="MIT License" src="https://img.shields.io/github/license/jellyfin/jellyfin-plugin-ldapauth.svg"/>
+</a>
+<a href="https://github.com/jellyfin/jellyfin-plugin-ldapauth/releases">
+<img alt="Current Release" src="https://img.shields.io/github/release/jellyfin/jellyfin-plugin-ldapauth.svg"/>
+</a>
+<a href="https://opencollective.com/jellyfin">
+<img alt="Donate" src="https://img.shields.io/opencollective/all/jellyfin.svg?label=backers"/>
+</a>
+<a href="https://features.jellyfin.org">
+<img alt="Feature Requests" src="https://img.shields.io/badge/fider-vote%20on%20features-success.svg"/>
+</a>
+<a href="https://forum.jellyfin.org">
+<img alt="Discuss on our Forum" src="https://img.shields.io/discourse/https/forum.jellyfin.org/users.svg"/>
+</a>
+<a href="https://matrix.to/#/+jellyfin:matrix.org">
+<img alt="Chat on Matrix" src="https://img.shields.io/matrix/jellyfin:matrix.org.svg?logo=matrix"/>
+</a>
+<a href="https://www.reddit.com/r/jellyfin">
+<img alt="Join our Subreddit" src="https://img.shields.io/badge/reddit-r%2Fjellyfin-%23FF5700.svg"/>
+</a>
+</p>
```

Comment:
We also probably don't need all those badges on the plugin repos?

Reply:
:thinking: agreed, yet which ones to keep?
proposed to keep: build status (given that the GHA build works), license, released version

Reply:
> GitHub Markdown really _really_ _**really**_ needs a way to center content, this is a mess...

Switching to adoc would allow the floating of images to a center position according to their doc, yet I have not tested that yet nor would that work for text unfortunately.

Reply:
This was mostly a complaint directed at GitHub Markdown and the unreadability of HTML, not at your particular change :sweat_smile:
I've made the exact same mess on other repos.

As for which badges to keep, I'd say the repo specific ones, the rest of them belong more on the main repositories I think.
But, if it looks good as-is right now, that's fine too. I'd take out the fider link however, as plugin feature request belong in the plugin issues the most.

Reply:
I did not take it as a comment directed at me, I just wanted to list an alternative since I have used adoc before :D

I already changed the badges to what I had proposed, so I'll leave it with that for now.